package-imports

```diff
@@ -142,8 +142,8 @@ sssd
 strace
 sudo
 swtpm
-#amd64 syslinux
-#amd64 syslinux-common
+syslinux
+syslinux-common
```

Comment:
curl -s https://packages.gardenlinux.io/gardenlinux/dists/experimental/main/binary-arm64/Packages.gz | gunzip | grep "Package: syslinux"
 
  Not sure why we have syslinux-common in arm64 experimental? 

assuming experimental was made with this run https://github.com/gardenlinux/repo/actions/runs/13146324922

Reply:
ah, because target arch of this package is architecture: all...